### PR TITLE
[EHV-2022] Remove and replace sessions

### PR DESCRIPTION
**Pull request by FloorD**
We had some cancellations, this is removing and replacing those sessions.

### CODE REVIEW

File: data/events/2022-eindhoven.yml [hunk 1]
```diff
@@ -325,7 +325,7 @@ program:
     date: 2022-10-21T00:00:00+02:00
     start_time: "12:30"
     end_time: "13:30"
-  - title: "vanessa-kantner"
+  - title: "Thatcher Peskens - On sustainability"
     type: talk
     date: 2022-10-21T00:00:00+02:00
     start_time: "13:30"
```
Comment (MikeRosTX) [inline data/events/2022-eindhoven.yml:331]: Want us to merge even though you're missing a speaker for this time slot 🤔❓

Comment (FloorD) [reply data/events/2022-eindhoven.yml:331]: Nope, not yet! That was why it was in draft, thanks @MikeRosTX ! Should have the info by tomorrow :)

Comment (FloorD) [reply data/events/2022-eindhoven.yml:331]: Thanks @mattstratton, I was waiting for some more info. Gots it now.

**Comment by phrawzty** [comment]
@FloorD Is this ready to be merged? If yes, you should be able to do it. If not, let @mattstratton know. :)

**Pull request merged**